srcpkgs/bash-completion/template

```diff
@@ -1,13 +1,18 @@
 # Template file for 'bash-completion'
 pkgname=bash-completion
 version=2.11
-revision=2
+revision=3
 build_style=gnu-configure
 make_install_args="profiledir=/etc/bash/bashrc.d"
 depends="bash"
+checkdepends="python3-pytest python3-pexpect"
 short_desc="Programmable completion for the GNU Bash shell"
 maintainer="skmpz <dem.procopiou@gmail.com>"
 license="GPL-2.0-or-later"
 homepage="https://github.com/scop/bash-completion"
 distfiles="https://github.com/scop/bash-completion/releases/download/${version}/bash-completion-${version}.tar.xz"
 checksum=73a8894bad94dee83ab468fa09f628daffd567e8bef1a24277f1e9a0daf911ac
+
+# Summary of tests: = 26 failed, 1106 passed, 457 skipped, 14 xfailed, 4 xpassed in 1126.01s (0:18:46) =
+# Let's try to fix this for 2.12; see #35201
```

Comment:
```suggestion
# Summary of tests: = 26 failed, 1106 passed, 457 skipped, 14 xfailed, 4 xpassed in 1126.01s (0:18:46) =
# FIXME: 26 tests fail, see #35201
```
`FIXME` is a common string, so people can find things to fix in the repository with a simple command `git grep FIXME`